tests/common/ixia/common_helpers.py

```diff
@@ -0,0 +1,95 @@
+"""This module contains some auxiliary functions that are required 
+to support automation activities. These functions are used for various 
+secondary activities like convert the ansible Unicode STDOUT output 
+to string, get IP address in a subnet, increment an IP address, get 
+VLAN subnet etc.
+
+This file is also a placeholder for auxiliary function that are
+required for supporting automation with Ixia devices in future:
+like collecting diagnostics, uploading and downloading files 
+to/from API server, processing the statistics after obtaining them
+in .csv format etc. 
+"""
+
+import ipaddr
+from netaddr import IPNetwork
+
+def incriment_ip_address (ip, incr=1) :
+    """
+    Increment IP address by an integer number.
+
+    Args: 
+       ip (str): IP address in string format.
+       incr (int):  Increment by.
+
+    Return:
+       IP address in the argument incremented by the given integer.
+    """
+    ipaddress = ipaddr.IPv4Address(ip)
+    ipaddress = ipaddress + incr
+    return_value = ipaddress._string_from_ip_int(ipaddress._ip)
+    return(return_value)
+
+
+def ansible_stdout_to_str(ansible_stdout):
+    """
+    The stdout of Ansible host is essentially a list of unicode characters.
+    This function converts it to a string.
+
+    Args: 
+        ansible_stdout: stdout of Ansible
+
+    Returns:
```

Comment:
Returns -> Return

Reply:
Google style docstring recommends “Returns” instead of “Return” https://google.github.io/styleguide/pyguide.html. So I kept it as it is.